solidus_product_assembly.gemspec

```diff
@@ -31,8 +31,8 @@ Gem::Specification.new do |s|
   end
 
   s.add_dependency 'deface'
-  s.add_dependency 'solidus_core', ['>= 1.0', '< 3']
-  s.add_dependency 'solidus_support', '~> 0.5'
+  s.add_dependency 'solidus_core', ['>= 1.0', '< 4']
+  s.add_dependency 'solidus_support'
```

Comment:
In the commit message you say:

> Also, it's using some code from `solidus_support` that was introduced in 0.8.1, so this should be the minimum version required. Set to 0.8 to losen restrictions.

But doesn't look like it's what has been done here.

Reply:
Updated gemspec in latest push!